react/components/SideBarItem.js

```diff
@@ -2,40 +2,55 @@ import React, { useState, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
 import { FormattedMessage } from 'react-intl'
-
-import { useRuntime } from 'vtex.render-runtime'
+import * as Amp from 'react-amphtml'
+import { useRuntime, Link } from 'vtex.render-runtime'
 import { IconMinus, IconPlus } from 'vtex.store-icons'
 
 import styles from '../categoryMenu.css'
 
+const getLinkParams = linkValues => {
```

Comment:
Links for subcategories like `pizza > See All` it's working. But it's not working for the other cases